### PR TITLE
Add my name

### PR DESCRIPTION
**Pull request by KiranSatyaRaj**
added my name to the readme file to wish Eddie.

### CODE REVIEW

**Review by kartikgajjar7 — CHANGES_REQUESTED** [review]
add line num 25 in the end

**Review by KiranSatyaRaj — COMMENTED** [review]
looks good, thanks

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by KiranSatyaRaj — COMMENTED** [review]
great, thanks for the help, hope this will be able to merge now

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

File: README.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ Syntax:
 |[Sreekar Shodhan](https://twitter.com/ShodhanSreekar)|Happy Birthday Eddie. I wish all your wishes come true and achieve your goals. Stay Healthy - Stay Happy|
 |[Kalyan Vurugonda](https://twitter.com/VurugondaKalyan)| Happy Birthday Eddie! 🥳🎉|
 |[Shubhangi](https://twitter.com/mshubhangi12)| Happy Birthday, Eddie! On your birthday, On your birthday, I wish you a lifetime of good health so that you can continue to inspire us and guide us during tumultuous times!! :heart_eyes: :balloon: :confetti_ball: :tada::sparkles:|
+|[Shubhangi](https://twitter.com/mshubhangi12)| Happy Birthday, Eddie! On your birthday, On your birthday, I wish you a lifetime of good health so that you can continue to inspire us and guide us during tumultuous times!! :heart_eyes: :balloon: :confetti_ball: :tada::sparkles:|
```
Comment (Pradumnasaraf) [inline README.md]:
```suggestion
```

**Review by Pradumnasaraf — APPROVED** [review]
Thank you!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**